site/content/features/sealedtypes.md

```diff
@@ -0,0 +1,506 @@
+---
+title: Sealed Classes (JEP 360)
+copyright: Cay S. Horstmann 2020. All rights reserved.
+jep: 360
+jdkversion: 15
+type: "sandbox"
+---
+
+
+Java 15 provides a preview of “sealed types”—types with a fixed set of direct subtypes. This feature allows for accurate modeling of type hierarchies that should not be open to arbitrary inheritance, and it allows the compiler to check for exhaustive pattern matching.
+
+## Controlling Subtypes
+
+Unless a class is declared `final`, anyone can form a subclass of it. What if you want to have more control? For example, suppose you feel the need to write your own JSON library because none of the ones out there do exactly what you need.
+
+The JSON standard says that a JSON value is an object, array, string, number, Boolean, or `null`. The Java language had, until now, no way of expressing that a `JSONValue` should be exactly one of those six types.
```

Comment:
Is "Boolean" upper case intentionally?

Reply:
Yes, it's an adjective derived from a person's name. https://english.stackexchange.com/questions/4481/should-the-word-boolean-be-capitalized. I'll change it to Boolean value.

Reply:
Ups, I wasn't aware of this. So please leave it like it is. (Another learning for me)